modules/gdscript/gdscript_warning.cpp

```diff
@@ -162,6 +162,12 @@ String GDScriptWarning::get_message() const {
 			return vformat(R"*(The default value is using "%s" which won't return nodes in the scene tree before "_ready()" is called. Use the "@onready" annotation to solve this.)*", symbols[0]);
 		case ONREADY_WITH_EXPORT:
 			return R"("@onready" will set the default value after "@export" takes effect and will override it.)";
+		case ACCESS_PRIVATE_MEMBER:
+			CHECK_SYMBOLS(1);
+			return vformat(R"(Trying to access a private member "%s" from an external place, which would cause problems during runtime.)", symbols[0]);
```

Comment:
It would be nice, but it's not exactly mandatory, to specify what it is instead of "member", similar to what's done in https://github.com/godotengine/godot/pull/104955 .

No need to change anything in this PR, it's for future reference